### ISSUE TITLE
App crashes after init

### ISSUE THREAD

**Issue by piernik** [admob-plus/admob-plus#296]
**Describe the bug**
App is building but after start it crashes.

**To Reproduce**
1. cmd `ionic start myApp tabs`
2. cmd `ionic cordova plugin add admob-plus-cordova --variable APP_ID_ANDROID="ca-app-pub-xxx"`
3. cmd `npm install @admob-plus/ionic`
4. cmd `ionic cordova run android --device --prod`

Build failed. Error is that I need to turn on `AndroidX`

5. Added line `<preference name="AndroidXEnabled" value="true" />` to `config.xml`
6. cmd `ionic cordova run android --device --prod`

Build failed. Error that `android build error: package android.support.annotation does not exist;`

7. cmd `ionic cordova plugin add cordova-plugin-androidx-adapter`

Build ok, but app crashes after opening. I did not changes any code in angular `src` folder - clean install.

**Expected behavior**
App should open without any ads.

**Screenshots**

**Environment**
`ionic@6.13.1`

package.json
```
{
  "name": "new",
  "version": "0.0.1",
  "author": "Ionic Framework",
  "homepage": "https://ionicframework.com/",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "ionic:device:build": "ionic cordova run android --device --prod"
  },
  "private": true,
  "dependencies": {
    "@admob-plus/ionic": "^1.1.0",
    "@angular/common": "~11.2.0",
    "@angular/core": "~11.2.0",
    "@angular/forms": "~11.2.0",
    "@angular/platform-browser": "~11.2.0",
    "@angular/platform-browser-dynamic": "~11.2.0",
    "@angular/router": "~11.2.0",
    "@ionic/angular": "^5.5.2",
    "cordova-android": "9.0.0",
    "rxjs": "~6.6.0",
    "tslib": "^2.0.0",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.1102.4",
    "@angular/cli": "~11.2.4",
    "@angular/compiler": "~11.2.0",
    "@angular/compiler-cli": "~11.2.0",
    "@angular/language-service": "~11.2.0",
    "@ionic/angular-toolkit": "^3.1.1",
    "@types/jasmine": "~3.6.0",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^12.11.1",
    "admob-plus-cordova": "^1.4.0",
    "codelyzer": "^6.0.0",
    "cordova-plugin-androidx-adapter": "^1.1.3",
    "cordova-plugin-device": "^2.0.2",
    "cordova-plugin-ionic-keyboard": "^2.2.0",
    "cordova-plugin-ionic-webview": "^4.2.1",
    "cordova-plugin-splashscreen": "^5.0.2",
    "cordova-plugin-statusbar": "^2.4.2",
    "cordova-plugin-whitelist": "^1.3.3",
    "jasmine-core": "~3.6.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~5.2.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-coverage-istanbul-reporter": "~3.0.2",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "protractor": "~7.0.0",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "~4.0.2"
  },
  "description": "An Ionic project",
  "cordova": {
    "plugins": {
      "cordova-plugin-whitelist": {},
      "cordova-plugin-statusbar": {},
      "cordova-plugin-device": {},
      "cordova-plugin-splashscreen": {},
      "cordova-plugin-ionic-webview": {
        "ANDROID_SUPPORT_ANNOTATIONS_VERSION": "27.+"
      },
      "cordova-plugin-ionic-keyboard": {},
      "admob-plus-cordova": {
        "APP_ID_ANDROID": "ca-app-pub-6477764423272591",
        "PLAY_SERVICES_VERSION": "19.8.0"
      },
      "cordova-plugin-androidx-adapter": {}
    },
    "platforms": [
      "android"
    ]
  }
}
```

`npx admob-plus info --clipboard`

```

  AdMob Plus Environment Info:
    System:
      OS: Windows 10 10.0.19042
      CPU: (4) x64 Intel(R) Core(TM) i5-4690 CPU @ 3.50GHz
      Memory: 2.85 GB / 15.95 GB
    Binaries:
      Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
      npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
    SDKs:
      Android SDK:
        API Levels: 26, 28, 29
        Build Tools: 26.0.2, 27.0.0, 28.0.3, 29.0.2, 30.0.1, 30.0.3
        System Images: android-26 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
    Languages:
      Java: 1.8.0_40 - C:\Program Files\Java\jdk1.8.0_40\bin\javac.EXE
    npmPackages:
      admob-plus-cordova: ^1.4.0 => 1.4.0 
      cordova-android: 9.0.0 => 9.0.0 
      cordova-plugin-androidx-adapter: ^1.1.3 => 1.1.3 
      cordova-plugin-device: ^2.0.2 => 2.0.2 
      cordova-plugin-ionic-keyboard: ^2.2.0 => 2.2.0 
      cordova-plugin-ionic-webview: ^4.2.1 => 4.2.1 
      cordova-plugin-splashscreen: ^5.0.2 => 5.0.2 
      cordova-plugin-statusbar: ^2.4.2 => 2.4.2 
      cordova-plugin-whitelist: ^1.3.3 => 1.3.3 
    Cordova:
      Version: 10.0.0
      Plugins:
        cordova-plugin-whitelist: ^1.3.3
        cordova-plugin-statusbar: ^2.4.2
        cordova-plugin-device: ^2.0.2
        cordova-plugin-splashscreen: ^5.0.2
        cordova-plugin-ionic-webview: ^4.2.1
        cordova-plugin-ionic-keyboard: ^2.2.0
        admob-plus-cordova: ^1.4.0
        cordova-plugin-androidx-adapter: ^1.1.3
    Ionic:
      Version: 6.13.1
    AndroidManifest.xml:
      <meta-data android:name="com.google.android.gms.ads.APPLICATION_ID" ... />: looks ok

```

**Additional context**

I'm making app that uses admob and google maps. I have problem (https://github.com/admob-plus/admob-plus/issues/250). So I tried fresh install, which came with problems above.

**Comment by ratson** [comment]
@piernik I run exactly the commands, no crash.

![screen](https://user-images.githubusercontent.com/2682937/114033017-7ab03880-98af-11eb-87e2-616ad88ce8cb.png)

**Comment by piernik** [comment]
@ratson Can You share Your project on github?

**Comment by ratson** [comment]
@piernik https://github.com/ratson/admob-plus-ionic-tabs-starter

**Comment by piernik** [comment]
@ratson thx. i'll analyse it tomorrow.

**Comment by piernik** [comment]
@ratson It's working. Thank You for Your time. Problem was with my `APP_ID_ANDROID`.
I think ionic docs  can be improoved by adding info about `<preference name="AndroidXEnabled" value="true" />` in `config.xml` file and installing `ionic cordova plugin add cordova-plugin-androidx-adapter`.

BTW. I also solved problem with `google-maps` plugin. 
I had to manually add
```
cordova.system.library.6=com.google.android.gms:play-services-maps:17.0.0
cordova.system.library.7=com.google.android.gms:play-services-location:17.0.0
```
In `platforms/android/project.properties`.

Once again thank You.
